tests/test_cluster.py

```diff
@@ -2450,6 +2450,7 @@ def try_delete_libs(self, r, *lib_names):
             except Exception:
                 pass
 
+    @pytest.mark.redismod
```

Comment:
You basically skipped those tests with `redismod` mark, because cluster tests exclude `redismod` https://github.com/redis/redis-py/blob/0d47d6527a10fb70f8fcfdf8df69ae3b11ec92ef/tasks.py#L73

It's not critical for these tests because T&F is deprecated anyway, but be careful with it in the future.